controllers/grid/queries/QueryNotesGridHandler.php

```diff
@@ -275,8 +273,8 @@ public function deleteNote($args, $request)
             return new JSONMessage(false);
         }
 
-        $noteDao->deleteObject($note);
-        return \PKP\db\DAO::getDataChangedEvent($note->getId());
+        $note->delete();
+        return \PKP\db\DAO::getDataChangedEvent($note->id);
```

Comment:
I would guess it's probably safe, but I haven't tried working with Eloquent model attribute data after calling a `delete()` operation. It might be worth specifically testing this to see if `$note->id` is still set after the `delete()`.

Reply:
Tested and confirmed that `$note->id` is still set.